templates/facet.template.hbs

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import { ILiFi } from "../Interfaces/ILiFi.sol";
+import { LibAsset } from "../Libraries/LibAsset.sol";
+import { LibSwap } from "../Libraries/LibSwap.sol";
+import { ReentrancyGuard } from "../Helpers/ReentrancyGuard.sol";
+import { SwapperV2 } from "../Helpers/SwapperV2.sol";
+import { Validatable } from "../Helpers/Validatable.sol";
+
+/// @title {{titleCase name}} Facet
+/// @author LI.FI (https://li.fi)
+/// @notice {{ description }}
+/// @custom:version 1.0.0
+contract {{titleCase name}}Facet is ILiFi, ReentrancyGuard, SwapperV2, Validatable {
+    /// Storage ///
+    
+    address public immutable example;
+
+
+    /// Types ///
+
+    /// @param exampleParam Example paramter
+    struct {{titleCase name}}Data {
+      string exampleParam;
+    }
+
+    /// Constructor ///
+
+    /// @notice Initialize the contract.
```

Comment:
imho not entirely accurate. 
The constructor does not and should not initialize the contract. I think this could be misleading. 
The constructor should only assign immutable params. Any other initialization (such as token lists, domainIDs, etc.) should be done in a separate init Method (that is missing here).

I think this pattern should also be carefully explained to third parties